packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -196,7 +196,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
     // If the returned value is not an array then check to see if it's a promise or PromiseLike. If it is then resolve it asynchronously.
     if (Array.isArray(suggestionsArray)) {
       this.resolveNewValue(updatedValue, suggestionsArray);
-    } else if (suggestionsPromiseLike.then) {
+    } else if (suggestionsPromiseLike) {
```

Comment:
I would agree, that would be the more correct fix.  Also - this is nitpicking, but the pickers folder inside components is the only one not capitalized.  Not a big thing but bugs my OCD.